src/open_inwoner/openzaak/tests/test_case_detail.py

```diff
@@ -877,6 +877,46 @@ def test_document_ordering_by_name(self, m):
         self.assertEqual(documents[1].name, "uploaded_document_title.txt")
         self.assertEqual(documents[2].name, "yet_another_document_title.txt")
 
+    @patch("open_inwoner.cms.cases.views.status.fetch_status_history")
+    def test_e_suite_missing_current_status(self, m, mock_fetch_status_history):
```

Comment:
Since the original issue (https://taiga.maykinmedia.nl/project/open-inwoner/issue/2037) is specifically about the upload button not showing (because `InnerCaseDetailView.is_file_upload_enabled_for_statustype` was returning `False` due to the statustype not being resolved) , I think it would be good to add checks for that here as well